src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js

```diff
@@ -28,6 +77,27 @@ class InvokeChiJi extends Analyzer {
   handleEnvelopingBreath(event) {
     this.envelopHealing += (event.amount || 0) + (event.absorbed || 0);
   }
+  
+  //stackbreakown management
+  handleApplyChijiStack() {
```

Comment:
From looking at logs this appears to never fire when you're at 3 stacks as you don't gain a stack in game.
You're going to have to track damage events from bok, sck, rsk, and totm bok to determin wasted stacks

Reply:
refactored to use damage events for stack management